src/lilURL.php

```diff
@@ -122,14 +123,23 @@ public function handlePOST($id = null, $user = null)
         $longurl = trim($_POST['theURL']);
 
         //Start by gathering all the GA items
-        if (!empty($_POST['gaSource'])) {
-            $gaTags = http_build_query([
+        if (isset($_POST['with-ga-campaign'])) {
+            $utmData =[
                 'utm_source' => $_POST['gaSource'],
                 'utm_medium' => $_POST['gaMedium'],
                 'utm_term' => $_POST['gaTerm'],
                 'utm_content' => $_POST['gaContent'],
                 'utm_campaign' => $_POST['gaName'],
-            ]);
+            ];
+
+            /*
+             * Verify GA data
+             */
+            if(!$this->validateGAData($utmData)){
```

Comment:
Please correct the conditional code style to match https://www.php-fig.org/psr/psr-2/#5-control-structures